### PR TITLE
feat: add start/end height flags to the fix cmd

### PR DESCRIPTION
**Pull request by huichiaotsou**
## Description

Add start/end height flags to the fix cmd

## Checklist
- [x] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit tests.  
- [x] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: cmd/fix/blocks/blocks.go [hunk 1]
```diff
@@ -55,7 +69,9 @@ func blocksCmd(parseConfig *parse.Config) *cobra.Command {
 		},
 	}
 
-	cmd.Flags().Bool(flagForce, false, "If set, forces the fetch of blocks by overwriting any existing one")
+	cmd.Flags().Bool(flagForce, false, "If set, forces the fetch of blocks by overwriting any existing ones")
```
Comment (MonikaCat) [inline cmd/fix/blocks/blocks.go]:
```suggestion
cmd.Flags().Bool(flagForce, false, "If set to true forces bdjuno to refetch all blocks from given height including overwriting any existing ones in database (default false)")
```

File: cmd/fix/blocks/blocks.go [hunk 2]
```diff
@@ -55,7 +69,9 @@ func blocksCmd(parseConfig *parse.Config) *cobra.Command {
...
+	cmd.Flags().Int64(flagStart, 0, "Set the height from which the refetching starts")
```
Comment (MonikaCat) [inline cmd/fix/blocks/blocks.go]:
```suggestion
	cmd.Flags().Int64(flagStart, 0, "Set the height to start refetching missing blocks from")
```

File: cmd/fix/blocks/blocks.go [hunk 3]
```diff
@@ -55,7 +69,9 @@ func blocksCmd(parseConfig *parse.Config) *cobra.Command {
...
+	cmd.Flags().Int64(flagEnd, 0, "Set the height to which the refetching finishes")
```
Comment (MonikaCat) [inline cmd/fix/blocks/blocks.go]:
```suggestion
	cmd.Flags().Int64(flagEnd, 0, "Set the height to finish refetching missing blocks at")
```

**Review by MonikaCat — APPROVED** [review]
tested, LGTM

**Review by rmontagnin — APPROVED** [review]

**Pull request merged**